bin/travis/test.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+source tools.sh
+
+bloopDir="bloop-server"
+[ -d "$bloopDir" ] && rm -rf "$bloopDir"
+mkdir "$bloopDir"
+
+bloopVersion=$(get_bloop_version)
+bloopURL="https://github.com/scalacenter/bloop/releases/download/v$bloopVersion/install.py"
+
+wget "$bloopURL"
+python install.py --dest $bloopDir
+
+$bloopDir/bloop ng-stop
```

Comment:
If those are dependant task, we should nto try to execute the second one if first one finishes with errors
```suggestion
$bloopDir/bloop ng-stop && $bloopDir/bloop server &>$bloopDir/bloop.log & 
```

It's worth considering this script to be something like:
```bash
install_bloop(){
  (...)
}
start_bloop(){
  (...)
}

install_bloop && start_bloop && sbt "$1"
```

Reply:
ng-stop only stops server if it's running - it should fail